### ISSUE TITLE
Document getting snapshots

### ISSUE THREAD

**Issue by DavidMGross** [ReactiveX/reactivex.github.io#172]
Original issue: ReactiveX/RxJava#3471

Simon Baslé
Oct 27 2015

While browsing around a bit I found out that snapshots were indeed published to JFrog's OSS Artifactory and could be added to a Maven project using the following repository configuration in the pom.xml:

```xml
<repository>
  <snapshots />
  <id>snapshots</id>
  <name>libs-snapshot</name>
  <url>http://oss.jfrog.org/artifactory/libs-snapshot</url>
</repository>
```

Maybe it would be worth it to add that information (plus instructions for Gradle/SBT/Ivy) in a "Getting Snapshots" section of the "Getting Started" wiki page?

**Comment by DavidMGross** [comment]
This concerns the portion of the RxJava docs that are still part of its own wiki, not part of reactivex.github.io (though at some point it might be nice to integrate the two).

**Comment by DavidMGross** [comment]
Closed via ReactiveX/RxJava#3471
